db_psql/seed.js

```diff
@@ -33,7 +33,11 @@ const users_queryString = `COPY trelia.users(user_name) FROM '${user_data_path}'
 const user_favorites_queryString = `COPY trelia.user_favorites(user_id, favorite_id) FROM '${user_favorites_data_path}' DELIMITER ',' CSV HEADER`;
 
 // seed the tables
-seed_table(listings_queryString, 'listings data successfully seeded');
-seed_table(similar_homes_queryString, 'similar_homes data successfully seeded');
-seed_table(users_queryString, 'user data successfully seeded');
-seed_table(user_favorites_queryString, 'user_favorites data successfully seeded');
+console.time('listings data seeded');
+seed_table(listings_queryString, console.timeEnd('listings data seeded'));
```

Comment:
and `console.timeEnd `be wrapped in an anonymous function?

Reply:
i ended up removing the console.time() invocations, because i couldn't get them to work properly, but good point -- wrapping in an anonymous function might be the solution.